libethereum/ClientBase.h

```diff
@@ -180,7 +180,7 @@ class ClientBase: public Interface
 	virtual void prepareForTransaction() = 0;
 	/// }
 
-	TransactionQueue m_tq;							///< Maintains a list of incoming transactions not yet in a block on the blockchain.
+	std::shared_ptr<TransactionQueue> m_tq;	///< Maintains a list of incoming transactions not yet in a block on the blockchain. This is never null.  Only passed on as a weak_ptr.
```

Comment:
Not sure if "Only passed as a weak_ptr" is true, `BlockChain::sync` receives it as shared_ptr

Reply:
I'm changing the sentence into "Only stored in this object."

Reply:
This was not changed yet